go.mod

```diff
@@ -6,63 +6,59 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v1.4.1
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/imdario/mergo v0.3.16
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
 	github.com/onsi/ginkgo v4.7.0-origin.0+incompatible
 	github.com/onsi/gomega v1.29.0
-	github.com/openshift/api v0.0.0-20240215031714-be926bb0d751
-	github.com/openshift/client-go v0.0.0-20240109161853-2425b4b6d3b3
-	github.com/openshift/cluster-kube-apiserver-operator v0.0.0-alpha.0.0.20240212125213-689a2bdc3773
-	github.com/openshift/cluster-openshift-apiserver-operator v0.0.0-alpha.0.0.20240125092135-74013916aadb
-	github.com/openshift/cluster-openshift-controller-manager-operator v0.0.0-alpha.0.0.20240204025224-1ba446b3a900
-	github.com/openshift/library-go v0.0.0-20231130112926-2285c05489d7
-	github.com/openshift/oc v0.0.0-alpha.0.0.20231101114527-035884cf4ed5
-	github.com/openshift/openshift-controller-manager v0.0.0-alpha.0
+	github.com/openshift/api v0.0.0-20240702171116-4b89b3a92a17
+	github.com/openshift/client-go v0.0.0-20240510131258-f646d5f29250
+	github.com/openshift/cluster-kube-apiserver-operator v0.0.0-alpha.0.0.20240725130911-de2f2ac2fc9a
+	github.com/openshift/cluster-openshift-apiserver-operator v0.0.0-alpha.0.0.20240426164535-d26f3002e10b
+	github.com/openshift/cluster-openshift-controller-manager-operator v0.0.0-alpha.0.0.20240621133439-bcc371ccbc9c
+	github.com/openshift/library-go v0.0.0-20240711192904-190fec8c3f09
+	github.com/openshift/oc v0.0.0-alpha.0.0.20240802093816-e8fb3c0574da
+	github.com/openshift/openshift-controller-manager v0.0.0-alpha.0.0.20240716164413-f0536caff579
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.16.0
-	github.com/sirupsen/logrus v1.9.0
+	github.com/prometheus/client_golang v1.17.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
-	k8s.io/api v0.29.4
-	k8s.io/apiextensions-apiserver v0.29.4
-	k8s.io/apimachinery v0.29.4
-	k8s.io/apiserver v0.29.4
-	k8s.io/cli-runtime v0.29.4
-	k8s.io/client-go v0.29.4
-	k8s.io/component-base v0.29.4
+	k8s.io/api v0.29.7
+	k8s.io/apiextensions-apiserver v0.29.7
+	k8s.io/apimachinery v0.29.7
+	k8s.io/apiserver v0.29.7
+	k8s.io/cli-runtime v0.29.7
+	k8s.io/client-go v1.5.2
```

Comment:
We want this to stay in line with the rest of the k8s deps, so it should be at `v0.29.7`